src/Provider/ImageProvider.php

```diff
@@ -145,7 +148,10 @@ public function getHelperProperties(MediaInterface $media, $format, $options = [
                     $media->getBox()->getWidth()
                 );
 
-                $params['srcset'] = implode(', ', $srcSet);
+                $params[$attrPrefix.'srcset'] = implode(', ', $srcSet);
+            } elseif ($attrPrefix) {
```

Comment:
Let's be explicit about the case we are referring to
```suggestion
            } elseif ($attrPrefix !== '') {
```